src/coreComponents/constitutive/contact/FrictionBase.hpp

```diff
@@ -153,9 +152,8 @@ class FrictionBaseUpdates
                                    arraySlice1d< real64 const > const & deltaDispJump,
                                    arraySlice1d< real64 const > const & penalty,
                                    arraySlice1d< real64 const > const & traction,
-                                   real64 const faceArea,
                                    arraySlice1d< real64 > const & tractionNew ) const
-  { GEOS_UNUSED_VAR( dispJump, deltaDispJump, penalty, traction, faceArea, tractionNew ); }
+  { GEOS_UNUSED_VAR( dispJump, deltaDispJump, penalty, traction, tractionNew ); }
```

Comment:
or this one? why not a pure virtual?

Reply:
because these are device functions. I am pretty sure that nvcc throws an error if we use a pure virtual. We do this a lot in our `kernelWrapper` objects (probably with the idea of getting nicer compilation errors) but, tbh, I start thinking that it would be better not to use `virtual` at all for these.